Makefile

```diff
@@ -1,18 +1,23 @@
+PATH := node_modules/.bin:$(PATH)
```

Comment:
or do `./node_modules/.bin/eslint` since I didn't have eslint globally which is what we have in babel https://github.com/babel/babel/blob/master/Makefile#L20